.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -101,6 +101,11 @@ jobs:
           echo "Transformer: $TAG_NAME"
           echo "User Transformer: $UT_TAG_NAME"
 
+      - name: Store Base Branch
+        id: store_base_branch
+        run: |
+          branch_name="origin/master"
```

Comment:
Two things:
- Even this is equal to hard coding the branch name. You need to extract the name right after you clone the repository.
- The syntax is not correct for storing variables. [Ref](https://github.com/rudderlabs/rudder-transformer/blob/b8b2ae7a63d07625c3057576ddb5915fe34ea926/.github/workflows/prepare-for-prod-deploy.yml#L40).

Reply:
updated the workflow